### ISSUE TITLE
Bower.json says 4.0.3, but JS file says 3.3.4

### ISSUE THREAD

**Issue by giovannibonetti** [danialfarid/ng-file-upload-bower#8]
Hi, there!
Thanks for the library! I find it very useful here in my startup.
I have just downloaded the release 4.0.3, and although the bower.json starts like this:

```javascript
{
  "name": "ng-file-upload",
  "main": "ng-file-upload.js",
  "version": "4.0.3",
  "homepage": "https://github.com/danialfarid/ng-file-upload",
}
```

The file angular-file-upload-all.js starts with:
```javascript
...
angularFileUpload.version = '3.3.4';
angularFileUpload.service('$upload', ['$http', '$q', '$timeout', function ($http, $q, $timeout) {
...
```

Hence the new features don't work...

**Comment by danialfarid** [comment]
Fixed in version 4.0.4
those files should've been removed.

**Comment by danialfarid** [comment]
Thanks for reporting
